### ISSUE TITLE
DXE-3243 Terraform provider is failing to create property without salt for verify_token_authorization behavior

### ISSUE THREAD

**Issue by sriraamas** [akamai/terraform-provider-akamai#481]
Hi there,.

### Terraform Version
1.5.0

### Affected Resource(s)
akamai_property_rules_builder

### Terraform Configuration Files
```hcl
    behavior {
      verify_token_authorization {
        algorithm           = "SHA256"
        escape_hmac_inputs  = true
        ignore_query_string = true
        key                 = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
        location            = "QUERY_STRING"
        location_id         = "__token__"
        use_advanced        = true
        transition_key = ""
        salt = ""
        failure_response    = true
      }
    }
```

### Expected Behavior
VerifyTokenAuthorization behavior should have been setup successfully with empty transition key and salt

### Actual Behavior
What actually happened?
Planning failed. Terraform encountered an error while generating this plan.

 ╷
│ Error: expected length of salt to be in the range (16 - 16), got 
│ 
│   with data.akamai_property_rules_builder.example_rule_default,
│   on rules.tf line 50, in data "akamai_property_rules_builder" "example_rule_default":
│   50:         ignore_query_string = true
│ 

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`

**Comment by majakubiec** [comment]
Hi @sriraamas,

Thank you for raising this issue.
We were able to reproduce it and created internal ticket to fix it.

**Comment by lkowalsk-akamai-com** [comment]
This issue was fixed in 5.4.0 release.
